erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -6,6 +6,20 @@ frappe.provide("erpnext.crm");
 
 cur_frm.email_field = "contact_email";
 frappe.ui.form.on("Opportunity", {
+
+	calculate_amount: function (frm) {
+		let total_amount = 0;
+		console.log(frm);
+		frm.doc.items.forEach(item => {
+			let amount = item.qty * item.rate;
+			console.log(amount);
+			frappe.model.set_value(item.doctype, item.name, 'amount', amount);
+			total_amount += amount;
+			amount = 0;
```

Comment:
Is this not re-initializing this variable on every loop iteration?

Reply:
sorry that 'amount = 0' by mistake is in loop.